Core/src/IfcGeometryConverter/RepresentationConverter.h

```diff
@@ -149,13 +149,6 @@ namespace OpenInfraPlatform {
 							BLUE_LOG(warning) << ex.what();
 							continue;
 						}
-						catch (const oip::InconsistentModellingException& ex)
-						{
-							// write the error to the console
-							BLUE_LOG(warning) << representation->getErrorLog() + ": Nothing is shown - sth wrong with IFC model.";
-							BLUE_LOG(warning) << ex.what();
-							continue;
-						}

```

Comment:
Because you've reverted the "merge" commit, this would get deleted. We don't want that.